tools/add_triage_signature.py

```diff
@@ -1010,45 +1167,53 @@ class ConsoleTimeoutSignature(ErrorSignature):
+        status_info = cluster["status_info"]
+        openshift_version = cluster["openshift_version"]
         report = ""
-        if ("waiting for console" in status_info and
-                "4.7" in openshift_version and
-                any(isVMware(host) for host in cluster['hosts'])):
+        if (
+            "waiting for console" in status_info
+            and "4.7" in openshift_version
+            and any(isVMware(host) for host in cluster["hosts"])
+        ):
             # ISO conversion according to
             # https://stackoverflow.com/questions/127803/how-do-i-parse-an-iso-8601-formatted-date#comment94022430_49784038
-            if datetime.fromisoformat(cluster['install_started_at'].replace('Z', '')) > datetime(2021, 5, 5):
-                report = "\n".join([
-                    "h2. Waiting for console timeout -possibly due to VMware hosts on OCP 4.7 ([bugzilla|https://bugzilla.redhat.com/1935539])-",
-                    "As of May 5th 2021, a fix has been pushed to production. That version contains the supposed fix. "
-                    "This means that the cause for the timeout in this ticket might not be due to this known bug - "
-                    "it might be caused by something else. Please investigate."
-                ])
+            if datetime.fromisoformat(cluster["install_started_at"].replace("Z", "")) > datetime(2021, 5, 5):
+                report = "\n".join(
+                    [
+                        "h2. Waiting for console timeout -possibly due to VMware hosts on OCP 4.7 ([bugzilla|https://bugzilla.redhat.com/1935539])-",
+                        "As of May 5th 2021, a fix has been pushed to production. That version contains the supposed fix. "
+                        "This means that the cause for the timeout in this ticket might not be due to this known bug - "
+                        "it might be caused by something else. Please investigate.",
+                    ]
+                )
             else:
-                report = "\n".join([
-                    "h2. Waiting for console timeout probably due to VMware hosts on OCP 4.7",
-                    "You can mark it as caused by issue [MGMT-4454|https://issues.redhat.com/browse/MGMT-4454]",
-                    "Solution will be provided via [bugzilla|https://bugzilla.redhat.com/1935539] ticket",
-                ])
+                report = "\n".join(
+                    [
+                        "h2. Waiting for console timeout probably due to VMware hosts on OCP 4.7",
+                        "You can mark it as caused by issue [MGMT-4454|https://issues.redhat.com/browse/MGMT-4454]",
+                        "Solution will be provided via [bugzilla|https://bugzilla.redhat.com/1935539] ticket",
+                    ]
+                )
 
         if report != "":
-            self._update_triaging_ticket(issue_key, report, should_update=should_update)
+            self._update_triaging_ticket(report)
```

Comment:
This one is so old... Can we maybe already remove it?

Reply:
Seems like its BZ was resolved so sure